libraries/AP_HAL_ChibiOS/hwdef/FlywooF745/defaults.parm

```diff
@@ -0,0 +1,17 @@
+# setup the four built-in Neopixel LEDs on chan 9
+SERVO9_FUNCTION 120
+NTF_LED_TYPES 257
+NTF_LED_LEN 4
+
+# enable compass 1
```

Comment:
default is all enabled...if you have additional external compasses of any type they will be detected but not used with this change...

Reply:
@Hwurzburg not sure about this board, but on the BeastF7 there is no external I2C and no CAN so no possibility of there ever being a compass

Reply:
it has the bus

Reply:
Yes, the reason this board is worth adding is that it has I2C pads. Now if they would make us an H7 version, but I digress...

Reply:
I removed the compass settings from the parm.